### ISSUE TITLE
dapple test no longer working

### ISSUE THREAD

**Issue by ethernomad** [nexusdev/dapple-test#6]
Using 0.8.6 with a patched ethereumjs-vm it works fine.

With 0.8.24 I get:
```
Testing...
/usr/lib/node_modules/dapple/node_modules/solc/soljson.js:1
(function (exports, require, module, __filename, __dirname) { var Module;if(!Module)Module=(typeof Module!=="undefined"?Module:null)||{};var moduleOverrides={};for(var key in Module){if(Module.hasOwnProperty(key)){moduleOverrides[key]=Module[key]}}var ENVIRONMENT_IS_WEB=typeof window==="object";var ENVIRONMENT_IS_WORKER=typeof importScripts==="function";var ENVIRONMENT_IS_NODE=typeof process==="object"&&typeof require==="function"&&!ENVIRONMENT_IS_WEB&&!ENVIRONMENT_IS_WORKER;var ENVIRONMENT_IS_SHELL=!ENVIRONMENT_IS_WEB&&!ENVIRONMENT_IS_NODE&&!ENVIRONMENT_IS_WORKER;if(ENVIRONMENT_IS_NODE){if(!Module["print"])Module["print"]=function print(x){process["stdout"].write(x+"\n")};if(!Module["printErr"])Module["printErr"]=function printErr(x){process["stderr"].write(x+"\n")};var nodeFS=require("fs");var nodePath=require("path");Module["read"]=function read(filename,binary){filename=nodePath["normalize"](filename);var ret=nodeFS["readFileSync"](filename);if

TypeError: Cannot read property 'contractAddress' of null
    at Object.web3.eth.getTransactionReceipt [as callback] (/usr/lib/node_modules/dapple/node_modules/dapple-test/lib/linklibraries.js:27:27)
    at /usr/lib/node_modules/dapple/node_modules/web3/lib/web3/method.js:142:25
    at /usr/lib/node_modules/dapple/node_modules/web3/lib/web3/requestmanager.js:89:9
    at engine.sendAsync (/usr/lib/node_modules/dapple/node_modules/dapple-chain/lib/server.js:185:11)
    at Web3ProviderEngine._inspectResponseForNewBlock (/usr/lib/node_modules/dapple/node_modules/web3-provider-engine/index.js:227:12)
    at /usr/lib/node_modules/dapple/node_modules/web3-provider-engine/index.js:129:14
    at /usr/lib/node_modules/dapple/node_modules/web3-provider-engine/node_modules/async/lib/async.js:52:16
    at /usr/lib/node_modules/dapple/node_modules/web3-provider-engine/node_modules/async/lib/async.js:269:32
    at /usr/lib/node_modules/dapple/node_modules/web3-provider-engine/node_modules/async/lib/async.js:44:16
    at async.eachSeries.resultObj.id (/usr/lib/node_modules/dapple/node_modules/web3-provider-engine/index.js:109:9)
```

**Comment by mhhf** [comment]
Fixed in 0.8.26
hmm, did this happened when you run dapple test on an internal chain?

**Comment by ethernomad** [comment]
It's fixed in 0.8.26. Was just running dapple test with default config.
